src/drivers/distance_sensor/cm8jl65/module.yaml

```diff
@@ -1,6 +1,27 @@
 module_name: Lanbao PSK-CM8JL65-CC5
 serial_config:
-    - command: cm8jl65 start -d ${SERIAL_DEV}
+    - command: cm8jl65 start -d ${SERIAL_DEV} -R p:SENS_CM8JL65_R_0
       port_config_param:
         name: SENS_CM8JL65_CFG
         group: Sensors
+
+parameters:
+    - group: Sensors
+      definitions:
+        SENS_CM8JL65_R_0:
+            description:
+                short: Distance Sensor Rotation
+                long: |
+                    Distance Sensor Rotation as MAV_SENSOR_ORIENTATION enum
+
+            type: enum
+            values:
+                25: ROTATION_DOWNWARD_FACING
+                24: ROTATION_UPWARD_FACING
+                12: ROTATION_BACKWARD_FACING
+                0: ROTATION_FORWARD_FACING
+                6: ROTATION_LEFT_FACING
+                2: ROTATION_RIGHT_FACING
+                100: ROTATION_CUSTOM
```

Comment:
The idea was to be able to specify the sensor rotation in any direction. This would be useful if you have a rig of distance sensor covering ~360 around the vehicle

Reply:
My question was how to specify/configure that?

Reply:
We would need to add another parameter in the driver to use the custom rotation

Reply:
Exactly. So I suggest to remove the `ROTATION_CUSTOM` from the config options for now.